app/controllers/admin/api/services/proxy/policies_controller.rb

```diff
@@ -33,4 +33,10 @@ def proxy_params
   def proxy
     @proxy ||= service.proxy
   end
+
+  def represent_on_error
+    return false unless proxy.errors.messages[:policies_config].include?(I18n.t(:invalid_policy, scope: 'activemodel.errors.models.policies_config.attributes.policies_config'))
+
+    :resource
+  end
```

Comment:
I would maybe change it to:

```
  def represent_on_error
    :resource if proxy.errors.messages[:policies_config].include?(I18n.t(:invalid_policy, scope: 'activemodel.errors.models.policies_config.attributes.policies_config'))
  end
```

it took me a while to understand what `false` means in this case.

A straight `if` seems a bit more easy to understand, and the behavior is the same - as the method would just return a `nil` (kind of equivalent to `false` in this context).

Reply:
Actually, I'm thinking...

Maybe it needs to be 
```
  def represent_on_error
    :resource if proxy.errors.messages[:policies_config] == [I18n.t(:invalid_policy, scope: 'activemodel.errors.models.policies_config.attributes.policies_config')]
  end
```

Otherwise, in a situation where there are both individual policy issues, and generic issues (e.g. policy chain size is too big, AND some policy is missing "version"), we will only represent the resource (because of the `include?`), so the user will not realize that there is a bigger problem with the chain.

So, with `==` we'd get something like 
```
{
    "errors": {
        "policies_config": [
            "contains some invalid policy",
            "is too long (maximum is 2097152 characters)"
        ]
    }
}
```

And we'll represent individual policies with their corresponding errors only when there are no generic issues.

Reply:
Also, I think this needs some inline comment. The PR comment is nice, but it will be gone, and I think it is worth leaving some lines on what are the consequences of this method.

Reply:
done!